cogs/command_utils/games_util.py

```diff
@@ -0,0 +1,396 @@
+import asyncio
+import random
+
+import discord
+from discord.ext import commands
+
+from cogs.utils import misc
+from cogs.utils.embeds import BaseEmbeds
+
+
+class RPSBase(BaseEmbeds):
+    def __init__(self, ctx, content, color=discord.Color.default(),
+                 title="Rock Paper Scissors!", rounds=3):
+        """Initializes and controls a embed with pagination."""
+        super().__init__(ctx, content, color=discord.Color.default(), title="Placeholder Title")
+
+        self.emb = discord.Embed(title=title, color=color)
+        self.content = content
+
+        self.react_controls = [
+            ("💎", self.rock),
+            ("📜", self.paper),
+            ("✂️", self.scissor),
+            ("\N{BLACK SQUARE FOR STOP}️", self.stop),
+        ]
+
+        self.current_round = 0
+        self.win_lose = [0, 0]  # index 0 is win | index 1 is lose
+        self.total_rounds = rounds
+        self.game_state = 0
+        self.meta_info = ""
+
+        self.emb.set_footer(text=f"Rounds: {self.current_round}/{self.total_rounds}\n"
+                                 f"{self.win_lose[0]} wins | {self.win_lose[1]} losses")
+
+    @staticmethod
+    async def rps_to_words(player_one, player_two):
+        conversion = str.maketrans({"1": "Rock", "2": "Paper", "3": "Scissor"})
+        return str(player_one).translate(conversion), str(player_two).translate(conversion)
+
+    async def is_playing(self):
+        if self.instance and self.current_round <= self.total_rounds:
+            return True
+        else:
+            return False
```

Comment:
R1703 : simplificable-if-statement
```suggestion
        return self.instance and self.current_round <= self.total_rounds
```